### ISSUE TITLE
can't copy image

### ISSUE THREAD

**Issue by TheRobertOne** [jpuri/react-draft-wysiwyg#470]
[#444](https://github.com/jpuri/react-draft-wysiwyg/issues/444)
version: 1.10.12
question: I test in storybook, can't copy image from Microsoft Word  also.

**Comment by belief-cyf** [comment]
yes!  i want to paste img to edit box, react-draft-wysiwyg can support this?

**Comment by jpuri** [comment]
This looks like regression issue, I will soon check.

**Comment by alexWang007** [comment]
copy img from website also see the error~i'm not sure is the same question。

**Comment by belief-cyf** [comment]
@alexWang007 you can find file from windows clipboard then post
![image](https://user-images.githubusercontent.com/19991842/30961036-83c68904-a477-11e7-9e86-76b3add775ec.png)
i just do this.
but maybe is not a good idea, This is just a temporary solution for me.

**Comment by nelsoneldoro** [comment]
for me at Linux this worked
```
handlePaste = () => {
    return false;
};
```

**Comment by nelsoneldoro** [comment]
This has worked 
https://github.com/jpuri/react-draft-wysiwyg/issues/498#issuecomment-461793404
